toolkit/.gitignore

```diff
@@ -2,8 +2,13 @@ out/
 
 # Temporary toolchain build files that are left behind after a failed build.
 scripts/toolchain/container/.bashrc
+scripts/toolchain/container/04-fix-sigstksz.patch
```

Comment:
You wouldn't know where these are coming from, would you? If they are temporary files required for the build, then I guess it might be better to redirect them to somewhere inside the `build` folder instead of having to update the `.gitignore` every time we add more of them.

Reply:
The latest ones came from #1623. They are copied into the container folder as part of the `create_toolchain_in_container.sh` script and are copied into the LFS tools folder in the Dockerfile. We utilize this mechanism when we want to ensure the patches used in our raw toolchain match the patches used in our official RPM builds. I do agree that it would be best to formulate a cleaner system for this, but for now I'm just looking to make things consistent.